server/src/test/java/org/eclipse/openvsx/search/DatabaseSearchServiceTest.java

```diff
@@ -336,6 +344,16 @@ private Extension mockExtension(String name, double averageRating, int ratingCou
 
     @TestConfiguration
     static class TestConfig {
+        @Bean
+        public MeterRegistry registry() {
+            return new SimpleMeterRegistry();
+        }
+
+        @Bean
+        public UrlConfigService urlConfigService() {
+            return new UrlConfigService();
+        }
+        
```

Comment:
Can't you add these two beans to the `@MockBean` list? These beans are not used in the test.